command_test.go

```diff
@@ -1862,6 +1862,84 @@ func TestAddGroup(t *testing.T) {
 	checkStringContains(t, output, "\nTest group\n  cmd")
 }
 
+func TestMissingGroupFirstLevel(t *testing.T) {
```

Comment:
I think this should be named "TestWrongGroupFirstLevel". This also applies to the other negative tests below.

Reply:
Done for all four new tests.